package.json

```diff
@@ -34,6 +34,7 @@
     }
   },
   "dependencies": {
+    "ejs-electron": "^2.1.1",
```

Comment:
thought: actually i don't think that another dependency is good idea. Let's try to use `preload.js` instead